python/ql/src/semmle/python/objects/ObjectAPI.qll

```diff
@@ -573,6 +573,9 @@ class ClassValue extends Value {
 abstract class FunctionValue extends CallableValue {
     abstract string getQualifiedName();
 
+    /** Gets a longer, more descriptive version of toString() */
+    abstract string descriptiveString();
```

Comment:
`descriptiveString` is already present in [master](https://github.com/Semmle/ql/blob/5a51d2cc4c18d34945c494f1c3ea6b3826e439bf/python/ql/src/semmle/python/objects/ObjectAPI.qll#L577) :|

So please remove that part from this PR. In the future I would recommend making a separate commit for such a thing.

Reply:
Resolved by merging master into this. Fortunately b/c of how git does things, the individual file versions go through merges nicely even if the commit histories are a little wonky, but we can always squash and merge to omit all those tedious details.